octodns/provider/route53.py

```diff
@@ -545,12 +545,21 @@ def health_checks(self):
         # We've got a cached version use it
         return self._health_checks
 
+    def _healthcheck_measure_latency(self, record):
+        return (
+            record._octodns.get('route53', {})
+            .get('healthcheck', {})
+            .get('measure_latency', True)
+        )
```

Comment:
It's a nit, but formatting wise I'd prefer the following here, the `()` caused me to have to look twice to see if it was returning a tuple.

```python
            return record._octodns.get('route53', {}) \
                .get('healthcheck', {}) \
                .get('measure_latency', True)
```